lib/minidoc/finders.rb

```diff
@@ -24,17 +24,16 @@ def exists?(selector = {})
 
     def find(id_or_selector, options = {})
       if id_or_selector.is_a?(Hash)
-        options.merge!(transformer: method(:wrap))
-        collection.find(id_or_selector, options)
+        collection.find(id_or_selector, options).lazy.map(&method(:wrap))
```

Comment:
The `transformer` feature seems to have been removed, which is what motivated this change. I haven't done any benchmarking, but I think this should be fine. Any concerns?